roles/converter-filters/tasks/main.yml

```diff
@@ -129,22 +129,43 @@
      - deploy_converter
      - setup
 
+- debug:
+    msg: "commands urls {{ converter_binary_url }}/{{ item }}"
+  with_items:
+    - 2224702/converti.zip
```

Comment:
strange naming

Reply:
it's the name assigned to binary files on github (see https://github.com/italia/docs-italia-comandi-conversione/releases/tag/v0.4 )

Reply:
La nuova release 0.4.1.2 usa un naming più sano :)